lib/improver/tests/nbhood/test_neighbourhoodprocessing.py

```diff
@@ -153,6 +152,60 @@ def set_up_cube(zero_point_indices=((0, 0, 7, 7),), num_time_points=1,
     return cube
 
 
+def set_up_cube_with_no_realizations(zero_point_indices=((0, 7, 7),),
+                                     num_time_points=1,
+                                     num_grid_points=16,
+                                     source_realizations=None):
```

Comment:
I think this relates to #50 as I think that there is a lot of duplication occurring with the setting up of cubes for unit tests, which could potentially be refactored out.